drizzle-orm/src/singlestore-core/columns/vector.ts

```diff
@@ -0,0 +1,77 @@
+import type { ColumnBaseConfig } from '~/column';
+import type { ColumnBuilderBaseConfig, ColumnBuilderRuntimeConfig, MakeColumnConfig } from '~/column-builder';
```

Comment:
should be `from '~/column-builder.ts'`
The reason why `attw` is failing - is because you didn't add `.ts`